src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { title, id } = tab;
+            const isNotCurrentTab = selectedTab !== tab;
```

Comment:
2. WE suggest you never compare the object with another object with strict comparison. It could produce a lot of bugs. Compare object using their unique identifier (id, title)

BAD EXAMPLE:
```jsx
if (cat === selectedCat) {
```

GOOD EXAMPLE:
```tsx
if (cat.id === selectedCat.id) {
```